### PR TITLE
Add OpenShift instructions and fix issues reported by QS tools

### PR DESCRIPTION
**Pull request by sgilda**

### CODE REVIEW

**Comment by sgilda** [comment]
I deployed this to OpenShift, and the application doesn't seem to render quite right on an iPhone. When I add a member, the 'Register' and 'Cancel' buttons are hidden under the footer menu so I can't add or cancel. The app is deployed here:  http://kitchensinkangularjstopcoat-sgilda.rhcloud.com/#/home

**Comment by sgilda** [comment]
I also managed to recreate the weird overlay issue on the iPhone by hitting the back and forward buttons in the browser.

File: kitchensink-angularjs-topcoat/README.md [hunk 1]
```diff
@@ -96,3 +96,123 @@ If you want to debug the source code or look at the Javadocs of any library in t
...
+
+### Test the OpenShift Application
+
+Now you will start to tail the log files of the server. To do this run the following command, remembering to replace the application name and login id.
+
+        rhc tail -a kitchensinkangularjstopcoat
+
+Once the app is deployed, you can test the application by accessing the following URL either via a browser or using tools such as curl or wget. Be sure to replace the `quickstart` in the URL with your domain name.
+
+    http://kitchensinkangularjstopcoat-quickstart.rhcloud.com/
```
Comment (rafabene) [inline kitchensink-angularjs-topcoat/README.md:199]: Change to http://kitchensinkangularjstopcoat-YOUR_DOMAIN_NAME.rhcloud.com/ as on previous lines

**Comment by rafabene** [comment]
Just a minor detail commented above

File: kitchensink-angularjs-topcoat/README.md (hunk 1 above)
Comment (sgilda) [inline kitchensink-angularjs-topcoat/README.md:199]: The OpenShift instructions in all the quickstarts work this way. We start out by telling them we use `quickstart` for the user/domain name. Then we remind them along the way.

Note the line above: 

```
    Be sure to replace the `quickstart` in the URL with your domain name.
```

Comment (rafabene) [inline kitchensink-angularjs-topcoat/README.md:199]: Right! I missed that :)

**Pull request merged**